### ISSUE TITLE
new server test fails when run against stage

### ISSUE THREAD

**Issue by jrgm** [mozilla/fxa-content-server#3549]
http://tests.dev.lcip.org:8111/viewLog.html?buildId=9332&buildTypeId=fxa_StageServerTests&tab=buildLog#_focus=18&state=18
```
[00:06:23]PASS: i18n - #get /i18n/client.json with language with two-part region with an unsupported region specified (456ms)
[00:06:23]PASS: i18n - #get /i18n/client.json with unsupported language returns default locale (342ms)
[00:06:24]PASS: i18n - #get /i18n/client.json with no locale returns default locale (342ms)
[00:06:24]FAIL: i18n - #get /503.html page - check text is rendered in dev mode (354ms)
[00:06:24]AssertionError: expected null to be truthy
[00:06:24]  at Assertion.<anonymous>  <node_modules/intern/node_modules/chai/chai.js:532:10>
[00:06:24]  at Assertion.Object.defineProperty.get  <node_modules/intern/node_modules/chai/chai.js:3515:29>
[00:06:24]  at Function.assert.ok  <node_modules/intern/node_modules/chai/chai.js:1990:31>
[00:06:24]  at Request.<anonymous>  <tests/server/l10n.js:293:14>
[00:06:24]  at Request.<anonymous>  <node_modules/intern/lib/Test.js:121:33>
[00:06:24]  at Request._callback  <node_modules/intern/lib/Test.js:134:23>
[00:06:24]  at Request.self.callback  <node_modules/request/request.js:198:22>
[00:06:24]  at Request.emit  <events.js:98:17>
[00:06:24]  at Request.<anonymous>  <node_modules/request/request.js:1035:10>
[00:06:24]  at Request.emit  <events.js:117:20>
[00:06:24]1/30 tests failed
```

**Comment by shane-tomlinson** [comment]
Hmm, something about the "dev mode" in "check text is rendered in dev mode" seems to suggest this test shouldn't be running when run against stage.

**Comment by shane-tomlinson** [comment]
Looking at the test though, I don't see why it should only run in dev mode, and what would cause the failure.

**Comment by shane-tomlinson** [comment]
@jrgm, if I visit https://accounts.stage.mozaws.net/503.html, it's a 404. It looks like the 502 and 503 pages aren't copied over in a place accessible by nginx.

**Comment by jrgm** [comment]
@shane-tomlinson - yeah, 50{0,2,3}.html were not copied into `dist` as on `latest`. But that's quite odd since both do `npm install && grunt build` IIRC.

**Comment by jrgm** [comment]
Odd. I looked at the build log for the stack without the 50{0,2,3}.html in place and:
```
DEBUG: Running "copy:error_pages" (copy) task
DEBUG: Copied 3 files
DEBUG: Done, without errors.
```
So, wtf?

**Comment by jrgm** [comment]
Crazy. There are the same set of `50{0,2,3}.html` on latest and stage, *except* in /dist on stage, but the log line above should mean they were copied out of dist/en, ya? @vladikoff?

**Comment by jrgm** [comment]
Aha! These files don't get copied into ./dist in a clean pull, npm install, grunt build locally either, despite the grunt copy:error_pages saying it copied 3 files (somewhere?).

**Comment by jrgm** [comment]
Hmm, copy:error_pages copies from `server/templates/pages/dist/en/50[023].html` to app.

Who's supposed to put these in `dist`?

**Comment by jrgm** [comment]
Ah, ok, order changed in grunttasks/build.js. It used to copy these files into `app`, then later copy to `dist`.
Now the copy to `dist` happens before these files have been put in `app`.

**Comment by shane-tomlinson** [comment]
> Now the copy to dist happens before these files have been put in app.

For whatever reason, the build step put those error pages into app instead of directly into dist. The error pages were then copied from app to dist. [All three error pages in .gitignore](https://github.com/mozilla/fxa-content-server/blob/ff8f0da0b9e87760d44fa9a83a0dfe56f069eb6b/.gitignore#L52) and these are not cleaned in our `grunt:clean` task. I had the 3 files in app already from a previous build, which ended up in dist as expected. With a fresh install, these 3 files would not already be in app, and would not be copied to dist.

Solution: 

```diff
diff --git a/grunttasks/copy.js b/grunttasks/copy.js
index d979397..1e6e42f 100644
--- a/grunttasks/copy.js
+++ b/grunttasks/copy.js
@@ -62,7 +62,7 @@ module.exports = function (grunt) {
     error_pages: { //eslint-disable-line camelcase
       files: [{
         cwd: '<%= yeoman.page_template_dist %>',
-        dest: '<%= yeoman.app %>',
+        dest: '<%= yeoman.dist %>',
         dot: true,
         expand: true,
         flatten: true,
```

If the server does not make use of the versions in `app`, we should also remove those 3 files from .gitignore while we are at it.

**Comment by jrgm** [comment]
Forgot to say "fixes #3549" in #3554. Merged and closing. Will file a followup issue to backport this to v0.57.0 so these error pages are in the right place on release.
